controllers/Frontend.php

```diff
@@ -1,61 +1,70 @@
 <?php
 
-require_once "models/Article.php";
-require_once "models/Comment.php";
-require_once "models/Form.php";
-
 class Frontend
 {
     public function home_cv()
     {
-        $model_form = new Form();
+        require_once "config/formulaires.php";
+        $errors = form_page_home_cv();
         
-        $page="frontend/home_cv";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('home_cv', 'frontend');
+        $myView->render(array('errors' => $errors));
     }
 
     public function legalnotice()
     {
-        $page="frontend/legalnotice";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('legalnotice', 'frontend');
+        $myView->render();
     }
     
     public function home()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
-
-        $page="frontend/home";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $manager = new ArticleManager();
+        $posts = $manager->get_posts_blog();
+        
+        $myView = new View('home', 'frontend');
+        $myView->render(array('posts' => $posts));
     }
 
     public function blog()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
+        $manager = new ArticleManager();
+        $posts = $manager->get_posts_blog_all();
         
-        $page="frontend/blog";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        $myView = new View('blog', 'frontend');
+        $myView->render(array('posts' => $posts));
     }
 
     public function post()
     {
-        $model_article = new Article();
-        $model_comment = new Comment();
-                
-        $page="frontend/post";
-        $topbar="frontend/topbar";
-        require "views/layout.php";
+        if(isset($_GET['id']))
+        {
+            $id = $_GET['id'];
+            $manager = new ArticleManager();
+            $posts = $manager->get_article_blog();
+            
+            $manager_user = new UserManager();
+            $admins = $manager_user->article_admin();
+            
+            $manager_comment = new CommentManager();
+            $responses = $manager_comment->get_comments_blog();
+            $errors = $manager_comment->form_comment_verification();
+        }
+        
+        if($posts == false){
+            header("Location:/page-erreur");
+        }
+        else{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method post uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them.](https://app.codacy.com/manual/JeromeLacquemant/projet-5/pullRequest?prid=5379676)